src/main/java/CycleTimeGUI.java

```diff
@@ -29,45 +40,89 @@ public CycleTimeGUI(int projectID,String authToken, String TAIGA_API_ENDPOINT,St
         this.firstDate = firstDate;
         this.lastDate = lastDate;
         System.out.println(projectID+"\n"+authToken+"\n"+TAIGA_API_ENDPOINT+"\n"+firstDate+"\n"+lastDate);
```

Comment:
not required

Reply:
Required as constructor is used to set data in getSprintData()

Reply:
Removed redundant initializations